Makefile

```
@@ -0,0 +1,56 @@
+	@go mod tidy
+	@go mod download
+	@echo "Dependencies installed!"
+
+# Run tests
+test:
+	@echo "Running tests..."
+	@go test ./...
+	@echo "Tests complete!"
+
+# Show help
+help:
```

checkmake (0.2.2)

[warning] Target body for "help" exceeds allowed length of 5 (14).

(maxbodylength)